dev/docker/kerberos-hive/hdfs-site.xml

```diff
@@ -0,0 +1,88 @@
+<configuration>
+  <property>
+    <name>dfs.replication</name>
+    <value>1</value>
+  </property>
+
+  <property>
+    <name>dfs.webhdfs.enabled</name>
+    <value>true</value>
+  </property>
+
+  <property>
+    <name>dfs.permissions.superusergroup</name>
+    <value>hdfs</value>
+  </property>
+
+  <property>
+    <name>dfs.block.access.token.enable</name>
+    <value>true</value>
+  </property>
+
+  <property>
+    <name>dfs.namenode.kerberos.principal</name>
+    <value>hdfs/mockhost@HADOOPKRB</value>
```

Comment:
Can this "mockhost" work?

Reply:
"mockhost" will become the real value.

<img width="867" alt="image" src="https://github.com/datastrato/gravitino/assets/15794564/7b2045e2-88c2-465b-99fa-b2e895b54f2d">